### PR TITLE
Bring in a collection of commits addressing the new location.

### PR DESCRIPTION
**Pull request by schmidtw**

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -230,23 +234,23 @@ func Provider() tfbridge.ProviderInfo {
...
+				fmt.Sprintf("github.com/pulumiverse/pulumi-%[1]s/sdk/", mainPkg),
 				tfbridge.GetModuleMajorVersion(version.Version),
 				"go",
 				mainPkg,
 			),
 			GenerateResourceContainerTypes: true,
 		},
 		CSharp: &tfbridge.CSharpInfo{
-			RootNamespace: "schmidtw",
+			RootNamespace: "pulumiverse",
```
Comment (ringods) [inline provider/resources.go:253]: .NET namespaces are usually capitalized. Can you update this?

**Review by ringods — CHANGES_REQUESTED** [review]

**Comment by ringods** [comment]
@schmidtw in `resources.go`, you map everything in `index` (constant: `mainMod`). Can you have a look if you can group a subset of the resources and functions into different modules?

Have a read up on this issue on what I mean with this:
https://github.com/pulumiverse/pulumi-heroku/issues/1

**Comment by schmidtw** [comment]
> @schmidtw in `resources.go`, you map everything in `index` (constant: `mainMod`). Can you have a look if you can group a subset of the resources and functions into different modules?
> 
> Have a read up on this issue on what I mean with this: [pulumiverse/pulumi-heroku#1](https://github.com/pulumiverse/pulumi-heroku/issues/1)

Yes and thanks for the suggestion.  I had initially split things out, but thought I was over-thinking the API since it was so small.  I'll adjust them.

**Comment by ringods** [comment]
@schmidtw I don't know about the VRA specifics, so I leave this mainly up to you. 

Though, I invite you to look at it from the perspective of the user of your VRA provider: what would make sense for this person when they know the VRA product terminology? Is the module and class naming "fluent" to use?

Changing this mapping later on results in breaking changes in the generated SDKs, which is something to prevent.

**Comment by schmidtw** [comment]
> @schmidtw I don't know about the VRA specifics, so I leave this mainly up to you.
> 
> Though, I invite you to look at it from the perspective of the user of your VRA provider: what would make sense for this person when they know the VRA product terminology? Is the module and class naming "fluent" to use?
> 
> Changing this mapping later on results in breaking changes in the generated SDKs, which is something to prevent.

I spent a fair bit of time trying to find the right combination of modules.  This ([5717df3](https://github.com/pulumiverse/pulumi-vra/pull/1/commits/5717df3dd0270b475f5400d387763c9d657daf83)) seems as reasonable as I can come up with & I'm sticking with that.

I was able to query a few things, create a VM and delete the VM so I think this code is working ok.  I'm completely new to VRA.

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  # NPM REGISTRY THEN ENSURE THE NPM_REGISTRY_URL IS CHANGED
+  NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
+  NUGET_FEED_URL: https://api.nuget.org/v3/index.json
+  PUBLISH_NUGET: true
+  # IF YOU NEED TO PUBLISH A PYPI PACKAGE THEN ENSURE AN PYPI_API_TOKEN
+  # SECRET IS SET AND PUBLISH_PYPI: TRUE. IF YOU WANT TO PUBLISH TO AN ALTERNATIVE
+  # PYPI REGISTRY THEN ENSURE THE PYPI_REPOSITORY_URL IS SET. IF YOU ARE USING AN API_TOKEN THEN
+  # YOU DO NOT NEED TO CHANGE THE PYPI_USERNAME (__token__) , IF YOU ARE USING PASSWORD AUTHENTICATION THEN YOU WILL
+  # NEED TO CHANGE TO USE THE CORRECT PASSWORD
+  PYPI_PASSWORD: ${{ secrets.PYPI_API_TOKEN }}
```
Comment (ringods) [inline .github/workflows/release.yml:27]: This needs to be `${{ secrets.PYPI_PASSWORD }}`

**Review by ringods — APPROVED** [review]

**Pull request merged**